<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color:rgb(0, 0, 0)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>370</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(170, 170, 255);
    color: rgb(0, 0, 0);

}


QPushButton:hover {
    background-color:rgb(128, 120, 215);
    color: rgb(0, 0, 0)
}

QPushButton:pressed {
    background-color: rgb(85, 170, 127);
    color: rgb(0, 0, 0)
}
</string>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>511</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox::title {
    color: rgb(255, 255, 255);
    subcontrol-origin: margin; /* Adjust the positioning of the title */
    padding: 0 5px; /* Optional: add space around the title */
}</string>
    </property>
    <property name="title">
     <string>Manager Information</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria Math</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
   color: rgb(170, 170, 255);
}</string>
     </property>
     <property name="text">
      <string>Email: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria Math</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
   color: rgb(170, 170, 255);
}</string>
     </property>
     <property name="text">
      <string>Name: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditname">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: rgb(0, 0, 0);
    border: 3px solid #777777; /* Optional: border color */
    padding: 5px; /* Optional: padding for better spacing */
}
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditemail">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: rgb(0, 0, 0);
    border: 3px solid #777777; /* Optional: border color */
    padding: 5px; /* Optional: padding for better spacing */
}
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditpassword">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: rgb(0, 0, 0);
    border: 3px solid #777777; /* Optional: border color */
    padding: 5px; /* Optional: padding for better spacing */
}
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria Math</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
   color: rgb(170, 170, 255);
}</string>
     </property>
     <property name="text">
      <string>Password: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria Math</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
   color: rgb(170, 170, 255);
}</string>
     </property>
     <property name="text">
      <string>Job Title:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria Math</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
   color: rgb(170, 170, 255);
}</string>
     </property>
     <property name="text">
      <string>Department:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria Math</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
   color: rgb(170, 170, 255);
}</string>
     </property>
     <property name="text">
      <string>Experience:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditposition">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>170</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: rgb(0, 0, 0);
    border: 3px solid #777777; /* Optional: border color */
    padding: 5px; /* Optional: padding for better spacing */
}
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditdepartment">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>220</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: rgb(0, 0, 0);
    border: 3px solid #777777; /* Optional: border color */
    padding: 5px; /* Optional: padding for better spacing */
}
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditexperience">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>270</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: rgb(0, 0, 0);
    border: 3px solid #777777; /* Optional: border color */
    padding: 5px; /* Optional: padding for better spacing */
}
</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>603</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
